bundles/core/org.eclipse.smarthome.core.thing.xml/thing-description-1.0.0.xsd

```diff
@@ -132,4 +133,18 @@
         </xs:simpleContent>
     </xs:complexType>
 
-</xs:schema>
+	<xs:complexType name="properties">
+		<xs:sequence>
+			<xs:element name="property" type="thing-description:property" minOccurs="1" maxOccurs="unbounded" />
+		</xs:sequence>
+	</xs:complexType>
+
+	<xs:complexType name="property">
+		<xs:simpleContent>
+			<xs:extension base="xs:string">
+				<xs:attribute name="name" type="xs:string" use="required" />

```

Comment:
Yes, I think we should recommend using camel case, but don't impose any restrictions in the xsd.